voorhoede/tudelft-spacefinder
# Refactor/remove unused filters and icons

# Changes


- refactor: remove ethernet and smartboard icons and filters

# Associated issue


Resolves:
https://trello.com/c/GFvL6BT1/71-remove-smartboard-filter-and-icon-from-spacefinder
https://trello.com/c/EvmPIhwg/69-remove-internet-cable-filter-and-icon-from-spacefinder

# How to test


1. Open preview link
2. Check if filters and icons are completely gone for ethernet and smartboard

# Checklist
